api/v1alpha1/releaseplan_types.go

```diff
@@ -53,6 +55,15 @@ type ReleasePlanSpec struct {
 
 // ReleasePlanStatus defines the observed state of ReleasePlan.
 type ReleasePlanStatus struct {
+	// Conditions represent the latest available observations for the releasePlan
+	// +optional
+	Conditions []metav1.Condition `json:"conditions"`
+
+	// ReleasePlanAdmission contains the namespaced name of the releasePlanAdmission this ReleasePlan is
```

Comment:
We don't really need the namespaced name as we have a target field, but that's ok to me.

Reply:
I kind of just copied the style of how we store references to CRs in CRDs. I'll leave it as is unless you want me to change it, which I am okay with